src/index.js

```diff
@@ -13,6 +13,9 @@ const rerenderEntireTree = (state) => {
 };
 
 rerenderEntireTree(store.getState());
```

Comment:
Consider making a separate function to hold `const state = store.getState();
  rerenderEntireTree(state);`
Then call it on 15th line instead of ` rerenderEntireTree(store.getState())` + pass its reference to store.subscribe.

Reply:
/fxd